schemas/eq_submission_schema.json

```diff
@@ -187,7 +190,27 @@
       "title": "Questionnaire ID",
       "default": "",
       "examples": [
-        "4782576236675773"
+        "7539657418968524"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "case_type": {
+      "$id": "#/properties/case_type",
+      "type": "string",
+      "title": "The Case_type Schema",
+      "default": "",
+      "examples": [
+        "HI"
```

Comment:
This field should be an enum. 

Runner validates it as: `validate=validate.OneOf(('HH', 'HI', 'CE', 'CI'))